src/Components/TodoList/ListItem/ListItem.tsx

```diff
@@ -1,9 +1,30 @@
-import React from 'react'
+import { Box, Button, Checkbox, Typography } from "@mui/material";
+import React from "react";
+import { COLORS } from "../../../colors";
 
-export default function ListItem() {
+interface ListItemProps {
+  description: string;
+}
+
+export default function ListItem({ description }: ListItemProps) {
   return (
-    <div>
-      list item
-    </div>
-  )
+    <Box sx={{ display: "flex", flex: 1, alignItems: "center" }}>
+      <Checkbox />
+      <Typography sx={{ display: "flex", width: "100%" }}>
+        {description}
+      </Typography>
+      <Button
+        variant="contained"
+        sx={{
+          bgcolor: COLORS.RED_BUTTON,
```

Comment:
It is best practice that when the sx gets too long we put it in the top of the file if possible, below the imports place a const called removeButtonSx ={...}